aws/README.md

```diff
@@ -1,36 +1,96 @@
+  - **SQS Queue** - SQS queue subscribed to CloudTrail SNS topic for Lacework  
+- **Lacework AWS CFG Integration** - Configures AWS CFG integration between Lacework and customer AWS account
+- **Lacework AWS CT Integration** - Configures AWS CT integration between Lacework and customer AWS account
+
+## Requirements
+Before you begin the following must be configured on the workstation running Terraform
+
+- [AWS API Access Key, Secret Access Key](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/)
+- [Terraform](terraform.io/downloads.html) `0.12.x`
+- [Lacework API Key](https://support.lacework.com/hc/en-us/articles/360011403853-Generate-API-Access-Keys-and-Tokens)
+
+## Usage
+
+### Enable New CloudTrail Configuration
+This example enables a new CloudTrail, IAM Role for Lacework, and then configures both integrations with Lacework
+
+```hcl
+provider "aws" {}
+
+provider "lacework" {}
+
+module "aws_config" {
+  source = "./modules/config"
+}
+
+module "aws_cloudtrail" {
+  source                = "./modules/cloudtrail"
+  bucket_force_destroy  = true
+  use_existing_iam_role = true
+  iam_role_name         = module.aws_config.iam_role_name
+  iam_role_external_id  = module.aws_config.external_id
+}
+```
+
+### Integrate Existing CloudTrail with Lacework
+This example uses an existing CloudTrail, S3 bucket, and SNS topic passed as inputs to the module. The example creates the SQS queue and IAM Role for Lacework, and then configures both integrations with Lacework
+
+```hcl
+provider "aws" {}
+
+provider "lacework" {}
+
+module "aws_config" {
+  source = "./modules/config"
+}
+
+module "aws_cloudtrail" {
+  source                = "./modules/cloudtrail"
+  use_existing_cloudtrail    = true
+  bucket_name                = "lacework-ct-bucket-8805c0bf"
+  sns_topic_name             = "lacework-ct-sns-8805c0bf
+  bucket_force_destroy       = true
+  use_existing_iam_role      = true
+  iam_role_name              = module.aws_config.iam_role_name
+  iam_role_external_id       = module.aws_config.external_id
+}
+```
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| bucket_force_destroy | Force destroy bucket (Required when bucket not empty) | `bool` | false | no |
+| bucket_name | Name of S3 bucket | `string` | lacework- | no |
+| cloudtrail_name | Name of existing cloudtrail | `string` | "lacework-cloudtrail" | no |
+| external_id_length | Length of External ID (max 1224) | `number` | 16 | no |
+| iam_role_external_id | External ID for IAM Role | `string` | "" | no |
+| iam_role_name |  The IAM role name | `string` | "lacework_iam_role" | no |
+| lacework_account_id | The Lacework AWS account that the IAM role will grant access | `string` | 434813966438 | no |
+| lacework_integration_name | The name of the integration in Lacework | `string` | TF config | no |
```

Comment:
One last thing; This variable is available for both modules, it is the name of the integration for config and cloudtrail, they both have defaults.

Reply:
👍  💯